src/commands/crashes/upload-symbols.ts

```diff
@@ -158,10 +171,16 @@ export default class UploadSymbols extends AppCommand {
 
   private validateParameters() {
     // check that user have selected either --symbol or --xcarchive
-    if (_.isNil(this.symbolsPath) && _.isNil(this.xcarchivePath)) {
-      throw failure(ErrorCodes.InvalidParameter, "specify either '--symbol' or '--xcarchive' switch");
+    if (_.isNil(this.symbolsPath) && _.isNil(this.xcarchivePath) && _.isNil(this.breakpadPath)) {
+      throw failure(ErrorCodes.InvalidParameter, "specify either '--symbol', '--xcarchive', or '--breakpad' switch");
     } else if (!_.isNil(this.symbolsPath) && !_.isNil(this.xcarchivePath)) {
       throw failure(ErrorCodes.InvalidParameter, "'--symbol' and '--xcarchive' switches are mutually exclusive");
+    } else if (!_.isNil(this.symbolsPath) && !_.isNil(this.breakpadPath)) {
+      throw failure(ErrorCodes.InvalidParameter, "'--symbol' and '--breakpad' switches are mutually exclusive");
+    } else if (!_.isNil(this.xcarchivePath) && !_.isNil(this.breakpadPath)) {
+      throw failure(ErrorCodes.InvalidParameter, "'--xcarchive' and '--breakpad' switches are mutually exclusive");
+    } else if (!_.isNil(this.breakpadPath) && !_.isNil(this.sourceMapPath)) {
+      throw failure(ErrorCodes.InvalidParameter, "'--breakpad' and '--sourcemap' switches are mutually exclusive");
```

Comment:
Checking this is true (since `--sourcemap` is not mutually exclusive to any of the other parameters).

Reply:
Sorry for not being clearer.

This is what I meant - but I wasn't sure if the check that breakpad and sourcemap are both being passed was correct (and intentional).

Reply:
Oh I'm sorry, I misunderstood -- that check is actually required. Sourcemaps (as currently implemented in the CLI) only pertain to iOS. Android sourcemaps (to the best of my understanding) would be implemented as "AndroidProGuard". Thus, this check is necessary to make sure the user isn't trying to upload iOS sourcemaps and Android symbols for the same app. I've added it back and resolved this conversation.

If you're think we're good to merge, I think we're good to merge! Just let me know. Otherwise, I'd be happy to discuss this further!